tools/devops/automation/templates/build/build.yml

```diff
@@ -432,7 +432,15 @@ steps:
     set -x
     set -e
 
-    make -C $(Build.SourcesDirectory)/xamarin-macios/tests package-tests
+    RC=0
+    make -C $(Build.SourcesDirectory)/xamarin-macios/tests package-tests || RC=$?
+    if [ $RC -eq 0 ]; then
+      echo "##vso[task.setvariable variable=macTestsPresent;isOutput=true]Succeeded"
+    else
+      echo "##vso[task.setvariable variable=macTestsPresent;isOutput=true]Failed"
+    fi
+    
```

Comment:
Maybe add this so that the step shows up as orange:

```
exit $RC
```

Reply:
Done